src/utils/privileges.cpp

```diff
@@ -99,11 +99,11 @@ static bool do_we_need_to_change_uid_or_gid() {
  * @pre Process must have CAPs needed for this special UID change: CAP_SETUID, CAP_SETGID, CAP_SETPCAP, CAP_CHOWN
  * @post User is not-root (not UID 0) or else exception is thrown
  */
-static void security_drop_root_from_sudo() {
+static bool security_drop_root_from_sudo() {
 	_fact("Dropping root (if we are root)");
```

Comment:
MOVE logic: I would rather move here the logic to read user's home getpwuid(....)  and return this as string (or empty sting if not user id change was done here).